tox.ini

```diff
@@ -15,7 +15,7 @@ extras =
 commands =
     python --version
     pip freeze
-    coverage run {envbindir}/pytest -xv []
+    pytest --cov {posargs:}
```

Comment:
You don't need `--cov` here if you have it in `setup.cfg`

Reply:
Yeah, I removed it from setup.cfg in 7c4fce2 when it failed to build in ec385af.